### DISCUSSION TITLE
Neustarts mit fos 8.0 [DECT panic]

### DISCUSSION THREAD

**Discussion by Conan179** [Freetz-NG/freetz-ng#1074] [Allgemein] [answered]
@fda77 Könntest du bitte mal kurz einen Blick auf diese Panic meldung werfen, warum die Box bei irgend was mit einer Mail einen neustart gemacht hat? Ich werde leider nicht schlau draus.
[panic.txt](https://github.com/user-attachments/files/17992978/panic.txt)

**Comment by fda77** [comment]
FOS 8 ist scheinbar buggy, obwohl es so lange gedauert hat und keine so berauschenden Dinge neu sind.

Für die 7530/7520 gibt es schon einen "plus" Hotfix
```
Verbesserungen in FRITZ!OS 08.00-116930
- System - Behoben: HÃ¤ufige Neustarts nach Update auf FRITZ!OS 8.00
```

https://service.avm.de/downloads/LaborPlus/fritzbox-7530/

--

Bei dir ist jedenfalls das Problem mit "dect", da sind Checksummen falsch
```
[DECTDRV_ERR] checksum error(len=31): calculated: 0x07bf != 0xffff (received)
```

das passiert ein paar mal und dann wird rebootet:
```
[  254.472975][  T740] [DECTDRV_ERR] PANIC!!!!
[  254.472978][  T740] Kernel panic - not syncing: PANIC: Disrupted communication to the STUB, probable reasons are: Mailchecksum wrong / Maillength wrong / missing Mail

...
[  254.488694][  T740] avm_set_reset_status: Soft-Reboot(PANIC)  - PANIC(2)UP(252)UTC(1733216825)FW(08.00)HW(233)HWS(8)BV(1.3614)BN(116067)FS(08.00)BT(1)BD(0)
```



Solltest du Kernelmodule von Freezt nutzen würde ich die mal zum testen weglassen, nicht dass die avm Sourcen nicht zum Kernel passen

**Reply by Conan179** [reply]
Es kommt vom DECT teil? ok das habe ich nicht gesehen gehabt, ich hab nur das mit der Zeile "mailchecksum wrong" gesehen.

Bewust nutzte ich keine kernel module, hier ist die .config 
[punktconfig_6591.txt](https://github.com/user-attachments/files/17993258/punktconfig_6591.txt)
Wen du da ein kernmodul siets, mache ich das gerne raus.

**Comment by fda77** [comment]
Scheinbar sind keine Kernelmodule ausgewählt, gut. Dann wird es wohl an AVM, die 7530 auch solche Problem hat, liegen.

Das "mail" kann auch die Kommunikation zwischen Komponenten wie Dect-Chip und Hauptprozessor sein. Der kernel sagt `Mailchecksum wrong` was weiter oben dect auch mit `checksum error` meinte.
Wie oft treten diesen Reboots denn auf? Hat darüber sonst noch niemand in Foren berichtet wie im IPPF oder https://www.vodafonekabelforum.de/viewforum.php?f=67 ?

Als workaround könnte man dect entfernen :)

**Reply by Conan179** [reply]
Ok, dann wird gedowngradet auf 7.57. Das einzige interessantes ist für mich, das in der Alltäglichen Pushnachricht jetzt die Auslastung des Kabelsegmends.
Das ist von einer ungefreetzte 6690
![segmentauslastung_031224](https://github.com/user-attachments/assets/a9b004af-0be7-4631-8a09-6f2c7f5ea56b)
So grob 1x pro 60min.
Ich danke dir für deine Hilfe :)

**Reply by fda77** [reply]
FOS 8.01 ist schon unterwegs: 30151c9d846859a7b2f2258b6e89d74388b601f0

**Comment by fda77** [comment]
Das ist doch eine praktische Funktion, ich lese zum 1. mal davon! Mit dem alten Docsis als man noch 4 Downstreamkanäle hatte, hatte ich  das mit einem DVB-C Stick an der Fritzbox und RRDstats auch geloggt: https://freetz-ng.github.io/freetz-ng/make/rrdstats.html#kabelsegment

Also es ist schon möglich dass es durch Freetz verursacht wird, ausschliessen kann man das nicht!
Aber offenbar ist der  FOS 8.0 Rollout gestoppt und die Top-Geräte 5690 und 7690 hängen auch noch im labor
https://github.com/Freetz-NG/freetz-ng/commits/master/docs/FIRMWARES.md

**Reply by Conan179** [reply]
Ich habe sie auch nur per zufall gesehen, das das jetzt in der Push Service dabei ist.
![segmentauslastung_031224a](https://github.com/user-attachments/assets/570a7a01-9f99-42bb-88d1-7544bd124225)
In den Settings steht es gar nicht dabei. O.O

**Reply by fda77** [reply]
Und im Browser kann man das nicht anzeigen lassen? Nur per mail?

**Reply by Conan179** [reply]
So wie es aussieht, momentan nur per Mail, im Webinterface, habe ich nichts gefunden.

**Comment by reimerp** [comment]
Hallo alle, leider ist auch in 8.01 der Fehler noch drin, aber es sieht nach einem AVM Problem aus. Freetz läuft stabil bis tatsächlich via DECT telefoniert wird. Mein FritzFon wurde mit 8.x auf 5.10 aktualisiert und Sprache beim telefonieren ist "abgehackt". Dazu passen die Logs ("PCM frame NOK", "checksum error")
```
[DECTDRV_ERR] PCM frame 347 NOK 04 00 23 14 18 f6 ff ff ff ff
[DECTDRV_ERR] checksum error(len=8): calculated: 0x0347 != 0xffff (received)

```
Das passiert bei mir wie bei @Conan179 ein paar mal, bevor der DECTDRV panic bekommt und die Box rebootet:
```
[13969.738713][ T1033] [DECTDRV] DRIVER side status of pcm link ...
[13969.745545][ T1033] [DECTDRV] tx_n_mails:              3090
[13969.751905][ T1033] [DECTDRV] rx_n_mails:              9557
[13969.758557][ T1033] [DECTDRV] rx_n_cs_errors:          27
[13969.764720][ T1033] [DECTDRV] rx_n_len_errors:         0
[13969.770787][ T1033] [DECTDRV] rx_n_mail_cnt_errors:    27
[13969.776966][ T1033] [DECTDRV] rx_n_lost_mails:         27
[13969.783160][ T1033] [DECTDRV] tx_max_len:              43
[13969.789502][ T1033] [DECTDRV] rx_max_len:              53
[13969.795662][ T1033] [DECTDRV_ERR] PANIC!!!!
```

Ich habe eine 6660 Cable, nutze keine Kernelmodule / Patches.
```
FritzOS 08.01 rev117720 (09.12.2024 23:22:01)
Freetz ng-25036M-3884ed5956 (26.12.2024 15:51:17)
```

**Reply by reimerp** [reply]
So, ich kann den Fehler nun re-produzieren, weiss aber noch nicht ob der auch im ORIGINAL Image auftaucht.
Einfach im FON ein Internet Radio auswählen. Ca alle 2-5 Sekunden stockt das Audio, und die Box loggt ca 20-30x:
`[DECTDRV_ERR] FP_MAC_SYNC_PCM_FRAMING, GOTYA!!!, pcm slot 1, sync 0x5a`
Irgendwann ist dann Schluss:

```
[122515.958906][ T1033] [DECTDRV_ERR] FP_MAC_SYNC_PCM_FRAMING, GOTYA!!!, pcm slot 1, sync 0x5a
[122515.968509][ T1029] Rx: 0x3 -> 0x3 (TranspModeMask: 0x2 MuteMask: 0x0)
[122515.968539][ T1029] #011[ 0] dect0 open  tdm-slotoffset(16bit):  0                           MUTE=0
[122515.968567][ T1029] #011[ 1] dect1 open  tdm-slotoffset(16bit):  1   DECT  TRANSP            MUTE=0
[122515.968588][ T1029] Tx: 0x3 -> 0x3 (TranspModeMask: 0x2 MuteMask: 0x0)
[122515.968605][ T1029] #011[ 0] dect0 open  tdm-slotoffset(16bit):  0                           MUTE=0
[122515.968627][ T1029] #011[ 1] dect1 open  tdm-slotoffset(16bit):  1   DECT  TRANSP            MUTE=0
[122515.968655][ T1029] PANIC:: (irq-cnt=30615876 spurious=13303 dma-reinit=0) cpu: 0 % (peak: 5 %)

```
Fazit DECT in 8.01 scheint kaputt

**Reply by fda77** [reply]
Du könntest versuchen das AVM zu melden. Aber ich geh davon aus dass die Support-Hansel 0-Ahnung haben und nur dumme Antworten zurückkommen. Ich hab AVM schon Jahre nichts mehr gemeldet da ich das als Zeitverschwendung ansehe :)

Oder versuch die noch nicht so ganz richtig veröffentlichte 8.02, möglicherweise läuft die besser: https://github.com/Freetz-NG/freetz-ng/discussions/1082

Es besteht natürlich auch die Möglichkeit dass dein Dect-Chip kaputt ist

Ich hab die Sourcen nach GOTYA durchsucht, gibts aber nicht: 
```
make kernel-unpacked
grep GOTYA source/kernel/ref-puma7.x86-6591_08.00/ -r
```

Dh diese Meldung komme aus dem AVM closed-source Wunderwerk.
Ein Bekannter meint mit closed-source will man entweder was verstecken oder peinliche Programmierkünste verheimlichen ^^

```
make firmware-nocompile
grep GOTYA build/original/filesystem/lib/modules/ -r
grep: build/original/filesystem/lib/modules/5.15.153/avm_dect/avm_dect.ko: Übereinstimmungen in Binärdatei
```

```
strings build/original/filesystem/lib/modules/5.15.153/avm_dect/avm_dect.ko | grep GOTYA -C5
HandsetNo == MailPtr->bHandsetNo
0[DECTDRV_ERROR_ASSERT]: %s at %s (%d)
3[DECTDRV_ERR] ApiFpFastUotaEnqueueTxData: nomem for %u (0x%x)
3[DECTDRV_ERR] API_FP_MAC_START_STOP_REQ %d -> %d
3[DECTDRV_ERR] other peer RFPI %*ph
3[DECTDRV_ERR] FP_MAC_SYNC_PCM_FRAMING, GOTYA!!!, pcm slot %u, sync 0x%x
3[DECTDRV_ERR] API_FP_UNITDATA_REQ: nomem
3[DECTDRV_ERR] API_FP_UNITDATA_REQ: ignore ill data length:%u
3[DECTDRV_ERR] !!! NO DECT RFPI SET !!! (please do DECT config of board, first)
3[DECTDRV_ERR] RFPI_IND, RFPI type %d, oFlags %x, inst %d, rfpi %*ph, ver %*ph, rFlags %x
3[DECTDRV_ERR] fpCap ext data queueSize %d, avmTpui %d/%d, etsiTpui %d/%d
```

**Comment by reimerp** [comment]
Ich glaube ich möchte gar nicht wissen, was AVM das zusammengefrickelt hat. Bei den ganzen ERRORs im syslog wundert mich, das überhaupt was geht :) -- Auf einer Original FW sieht man das ja auch nicht. 
Ich bin kurz zurück auf 7.57 (gefreezt) und DECT läuft anstandslos (Telefon und auch Internetradio) -- keine PCM oder DECT errors.  Dh DECT HW schein ok und auch das FON Update wars nicht. 7.57 läuft ja aber noch auf 4.9.279 kernel. Für 8.0 (.1 oder 2) original hatte ich noch keine Zeit/Lust weil ich auf diverse Freetz Addons angewiesen bin, vielleicht warte ich noch auf offizielle 8.0.2, hoffentlich ab 06. Januar....

**Reply by fda77** [reply]
Es gibt wirklich faszinierende Fehler die AVM so mit der Zeit erfunden hat.
Zb bei der 7390 und 7490 (7590 und neuer nicht getestet) die ich im Router-Modules Internet-via Lan1/Wan benutzte, haben den kompletten Internettraffic auf ich glaub es waren 30% (von zb 100mbit) beschränkt,  sobald eine ISDN-Telefon von der Gabel genommen wurde.
Dies mache das PA (packet accelerator) Meisterwerk um genug Bandbreite für voip freizubekommen. Richtig freigegeben wurde es das nicht mehr. Daraus entstanden denn Kernelpatches wie https://github.com/Freetz-NG/freetz-ng/blob/master/make/kernel/patches/3.10.73/7490_06.93/902-PA_noloadcontrol.diff
Mittlerweile hat mein Internetrouter keine AVM-Software mehr, die ist mir zu schlecht :-D

**Comment by reimerp** [comment]
kurzes Update zu 8.0.2 (freetz): erneut auch ein Update meines FONs dabei. Die DECT Fehler sind noch da aber scheinen etwas weniger geworden zu sein. Dennoch kommt es zu Audio Aussetzern und im Log zu den Mailchecksum Fehlern. Mittlerweiler wird man wenigstens gewarnt:
```
[DECTDRV_ERR] Disrupted communication to the STUB, probable reasons are: Mailchecksum wrong / Maillength wrong / missing Mail - ErrorCount=8 - last pcmctrl-handling before 0 ms
[DECTDRV_ERR] If next time the error count exceeds the threshold value, a panic will be triggerd.
```
und tatsächlich kommt die PANIX dann auch bald....

Noch keine Zeit gehabt DECT im Original Image zu testen, wie gesagt ich brauche dnsmasq, hab keine logs/shell access im Original und der Weg zurück zu freetz hemmt mich.

Rein vom Log her ist ja
`checksum error(len=19): calculated: 0x0f05 != 0xffff (received)` die remote Prüfsumme 0xffff und verkehrt, und diese kommt ja vom STUB / ARM Teil, was ja durch freetz nicht gepatcht wird...

**Reply by fda77** [reply]
Es gab schonmal eine Fritzbox bei der Freetz nichts ins syslog(??) schreiben durfe weil dann dect nicht ging.
Am besten würdest du mal die ganzen alten Tickets im Trac (des alten svns) dazu lesen.
Evtl würde die Lösung dazu auch bei dir helfen?
hippe hatte auf boxmatrix eigentlich eine kopie des trac, aber das ist wohl hinüber: `https://boxmatrix.info/wiki/Trac-Server`

Vielleicht war das dieser Patch: https://github.com/Freetz-NG/freetz-ng/blob/master/patches/scripts/180-patch_printk.sh
Heute gibt es aber init.d nicht mehr so oft und avm hat das meiste zum system-nachbau umgeschrieben

Aber bevor man da das mach, probier zuerst mal ohne freetz.

**Reply by Conan179** [reply]
@reimerp hast du es mit der 8.03er fw schon probiert, ob es noch auftritt?

**Reply by reimerp** [reply]
Ja, Problem tritt noch auch, wenn es  sich auch etwas verbessert hat: weniger Mailchecksum Fehler im Logs und akustisch weniger Drops, aber scheinbar gibts ein Maximum ab dem die Box die Grätsche macht. Ich habe allerdings noch nicht die Original FW probiert. Ich brauche einige Freetz Features sodass ich wieder auf 7.58 bin

**Reply by Conan179** [reply]
Ok, danke.

**Reply by Conan179** [reply]
@fda77 Ist patch noch drin? Wie kann ich ihn deaktiveren, um zu testen ob es daran liegt? Ich plane auf die neuste fw zu updaten.

**Reply by Conan179** [reply]
@fda77 also ich hab ein frisches freetz ausgecheckt, den patch https://github.com/Freetz-NG/freetz-ng/blob/master/patches/scripts/180-patch_printk.sh gelöscht.
Der fehler tritt immer noch auf auch bei der 8.10 Labor ich bräuchte bitte mal hilfe.

**Reply by aus-sua** [reply]
>  ich bräuchte bitte mal Hilfe

Ich bin zwar nicht fda77, aber ohne alle Deine jetzigen aktuellen Logs/errors/panics/usw. und 
ohne Deine aktuelle System-configs usw. wird Dir keiner helfen können.
Stelle die bitte doch alle hier ein...

PS:
Ich habe/hatte eine 5590, dort ist das Problem wohl ähnlich.
Bei DECT-Gesprächen wurde die Qualität des jeweiligen VoIP-Gespräches immer
schlechter, dann bootete die 5590er neu, machmal auch direkt bei Gesprächsbeginn.
Die Box ist z.Z. für mich nicht erreichbar, die Box ist "produktiv" im Einsatz, deshalb
hat die z.Z. nun ein Original-AVM-FOS v.8.02, da gibt es diese Probleme nicht.
Logs etc. kann ich damit z.Z. nicht liefern, ggf. in ca. zwei/drei Wochen wieder, wenn ich
an die Box komme und freetz-NG wieder aktiviere...

**Reply by Conan179** [reply]
[punktconfig.txt](https://github.com/user-attachments/files/21167173/punktconfig.txt)
[panic.txt](https://github.com/user-attachments/files/21167203/panic.txt)
`2025-07-10T18:36:46+02:00 192.168.1.1 kernel: [ 1230.789922][  T744] trigger too late:: (irq-cnt=299344 spurious=319 dma-reinit=0) cpu: 0 % (peak: 3 %)
2025-07-10T18:36:46+02:00 192.168.1.1 kernel: [ 1230.800586][  T744] avm_DebugSignal: 0 kernel info: 0 [pcmlink_ctrl/1]
2025-07-10T18:36:49+02:00 192.168.1.1 kernel: [ 1233.739619][  T754] [DECTDRV_ERR] no STATUS from STUB received, probably dead - last pcmctrl-handling before 1 ms
2025-07-10T18:36:49+02:00 192.168.1.1 kernel: [ 1233.751349][  T754] avm_DebugSignal: 0 kernel info: 0 [avm_dect_thread]
2025-07-10T18:36:49+02:00 192.168.1.1 kernel: [ 1233.759042][  T754] [DECTDRV_ERR] Disrupted communication to the STUB, probable reasons are: Mailchecksum wrong / Maillength wrong / missing Mail - ErrorCount=35 - last pcmctrl-handling before 0 ms
2025-07-10T18:36:49+02:00 192.168.1.1 kernel: [ 1233.778867][  T754] [DECTDRV] STUB side status of pcm link ...
2025-07-10T18:36:49+02:00 192.168.1.1 kernel: [ 1233.785614][  T754] [DECTDRV] n_tx:                    892
2025-07-10T18:36:49+02:00 192.168.1.1 kernel: [ 1233.791867][  T754] [DECTDRV] n_rx:                    292
2025-07-10T18:36:49+02:00 192.168.1.1 kernel: [ 1233.798140][  T754] [DECTDRV] n_rx_cs_errors:          1
2025-07-10T18:36:49+02:00 192.168.1.1 kernel: [ 1233.804166][  T754] [DECTDRV] n_rx_len_errors:         0
2025-07-10T18:36:49+02:00 192.168.1.1 kernel: [ 1233.810117][  T754] [DECTDRV] n_rx_overflows:          0
2025-07-10T18:36:49+02:00 192.168.1.1 kernel: [ 1233.816366][  T754] [DECTDRV] n_rx_buffer_overruns:    0
2025-07-10T18:36:49+02:00 192.168.1.1 kernel: [ 1233.822352][  T754] [DECTDRV] n_phase_info_errors:     0
2025-07-10T18:36:49+02:00 192.168.1.1 kernel: [ 1233.828320][  T754] [DECTDRV] n_dsp_int_errors:        0
2025-07-10T18:36:49+02:00 192.168.1.1 kernel: [ 1233.834296][  T754] [DECTDRV] n_dsp_overflows:         0
2025-07-10T18:36:49+02:00 192.168.1.1 kernel: [ 1233.840237][  T754] [DECTDRV] n_mail_overflow:         0
2025-07-10T18:36:50+02:00 192.168.1.1 kernel: [ 1233.846424][  T754] [DECTDRV] phase_info:              138
2025-07-10T18:36:50+02:00 192.168.1.1 kernel: [ 1233.852703][  T754] [DECTDRV] n_lost_messages:         1
2025-07-10T18:36:50+02:00 192.168.1.1 kernel: [ 1233.858672][  T754] [DECTDRV]
2025-07-10T18:36:50+02:00 192.168.1.1 kernel: [ 1233.862112][  T754] [DECTDRV] DRIVER side status of pcm link ...
2025-07-10T18:36:50+02:00 192.168.1.1 kernel: [ 1233.868872][  T754] [DECTDRV] tx_n_mails:              294
2025-07-10T18:36:50+02:00 192.168.1.1 kernel: [ 1233.875045][  T754] [DECTDRV] rx_n_mails:              878
2025-07-10T18:36:50+02:00 192.168.1.1 kernel: [ 1233.881197][  T754] [DECTDRV] rx_n_cs_errors:          22
2025-07-10T18:36:50+02:00 192.168.1.1 kernel: [ 1233.887308][  T754] [DECTDRV] rx_n_len_errors:         0
2025-07-10T18:36:50+02:00 192.168.1.1 kernel: [ 1233.893329][  T754] [DECTDRV] rx_n_mail_cnt_errors:    24
2025-07-10T18:36:50+02:00 192.168.1.1 kernel: [ 1233.899380][  T754] [DECTDRV] rx_n_lost_mails:         24
2025-07-10T18:36:50+02:00 192.168.1.1 kernel: [ 1233.905464][  T754] [DECTDRV] tx_max_len:              59
2025-07-10T18:36:50+02:00 192.168.1.1 kernel: [ 1233.911526][  T754] [DECTDRV] rx_max_len:              53
2025-07-10T18:36:50+02:00 192.168.1.1 kernel: [ 1233.917782][  T754] [DECTDRV_ERR] PANIC!!!!

Message from syslogd@192.168.1.1 at Jul 10 18:36:50 ...
 kernel: [ 1233.922503][  T754] Kernel panic - not syncing: PANIC: Disrupted communication to the STUB, probable reasons are: Mailchecksum wrong / Maillength wrong / missing Mail
2025-07-10T18:36:50+02:00 192.168.1.1 kernel: [ 1233.922503][  T754] Kernel panic - not syncing: PANIC: Disrupted communication to the STUB, probable reasons are: Mailchecksum wrong / Maillength wrong / missing Mail
`
von meiner 6690 fw 8.03


und eine panic von meiner 6591
[panic_6591.txt](https://github.com/user-attachments/files/21167442/panic_6591.txt)
Die 6690 ist mash master
die 6591 mesh client
an der 6690 hängt eine 7272 (uralt fw)
und eine 6590 mit fw 7.20 (kein update geplant)

**Reply by bodop** [reply]
> > ich bräuchte bitte mal Hilfe
> 
> Ich bin zwar nicht fda77, aber ohne alle Deine jetzigen aktuellen Logs/errors/panics/usw. und ohne Deine aktuelle System-configs usw. wird Dir keiner helfen können. Stelle die bitte doch alle hier ein...
> 
> PS: Ich habe/hatte eine 5590, dort ist das Problem wohl ähnlich. Bei DECT-Gesprächen wurde die Qualität des jeweiligen VoIP-Gespräches immer schlechter, dann bootete die 5590er neu, machmal auch direkt bei Gesprächsbeginn. Die Box ist z.Z. für mich nicht erreichbar, die Box ist "produktiv" im Einsatz, deshalb hat die z.Z. nun ein Original-AVM-FOS v.8.02, da gibt es diese Probleme nicht. Logs etc. kann ich damit z.Z. nicht liefern, ggf. in ca. zwei/drei Wochen wieder, wenn ich an die Box komme und freetz-NG wieder aktiviere...

Bist Du an der Stelle weiter gekommen? Ich habe auch die 5590, die ohne Freetz wunderbar telefoniert. Ich bin mir schon nicht sicher, wie man an die panic.txt kommt. Muss Freetz geflasht sein (nicht nur "push_firmware -afu"), damit man nach dem Neustart aus der Freetz-UI die panic.txt ziehen kann? Hast Du den Hinweis mit dem "supervisor" verstanden?

Freetz macht ja einen tollen Eindruck und hat für mich wichtige Erweiterungen. Nur telefonieren sollte man können. Der Rest des Haushalts ist nicht so erfreut, wenn ich ständig das Telefon kaputt mache.

**Reply by bodop** [reply]
Mit 8.20 läuft es jetzt über eine Woche stabil.

**Reply by reimerp** [reply]
Das freut mich zumindest für Dich. Ich war schon im November 25 auf 8.21. Es wurde besser, aber entgültig weg war die panic nicht. Sporadische Aussetzer beim telefonieren, aber ganz einfach lässt sich das Problem forcieren indem über ein Dect Handgerät Internetradio gehört wird, dann passiert die Panic nach 20-30 Sekunden. Bin wieder auf 7.57 und was neueres als 8.21 hab ich für 6660 noch nicht gesehen

**Reply by reimerp** [reply]
Ich kann mir nicht vorstellen das es an irgeneinem Patch liegt. Der ganze DECT Kram ist nicht im Atom sondern im ARM Teil der Box , da kommen auch die Meldungen her und der wird m.W nicht gepatcht.

**Reply by fda77** [reply]
>  ARM Teil .. nicht gepatcht.

ja die 2 arm partitionen werden einfach unverändert übernommen

**Reply by Conan179** [reply]
@reimerp Auf meiner Arbeit habe ich eine 6690 stehen, mit einem Dect Handteil und 5 FRITZ!Smart Energy 200, der letzte neustart war am 10.02 mit fw 8.21.
Zuhause eine 6690 mit einem DECT Handteil, die hat dect panics, der unterschied ist nur, meine hat freetz, die auf der Arbeit nicht.

**Reply by fda77** [reply]
Angenommen du hast eine lila und aquamarin-blaue Fritzbox.
Die lila wird vom Auto überfahren.
Ist die Farbe schuld?

**Reply by Conan179** [reply]
Also eine 6690 mit freetz, eine 6591 mit freetz machen reboot, eine 6690 ohne freetz nicht und viele andere.
reimerp hat das problem auch.

**Reply by fda77** [reply]
Passiert auch ohne freetz: https://www.reddit.com/r/fritzbox/comments/1fnfhnx/6690_cable_im_restart_loop/

**Reply by Conan179** [reply]
"Power/Cable leuchtet, dann leuchten alle, dann geht alles aus und wieder von vorne. " 
Naja, wen es der selbe fehler wie hier wäre, ja passiert auch ohne freetz.

**Reply by reimerp** [reply]
das Problem mit den Neustarts im Original Image ist ja leider: Die GUI sagt dir nicht warum das passiert ist, es sei denn Du hast telnet freigeschaltet und kannst dir zB die DECT logs per dmesg ansehen...

**Comment by fda77** [comment]
Tritt das denn überhaupt ohne Freetz auf? Oder ist das so "normal"

**Reply by Conan179** [reply]
Nein. Auf meiner Arbeit steht eine ungefreetzte 6690 die seit dem 02.06.26 ohne neustarts durch läuft.

**Reply by fda77** [reply]
Dann pass den Patch an dein Gerät an. und schau ob die Dateien überhaupt noch  rc.S und S17-isdn und S11-piglet heissen, es gab da noch keinen supervisor

**Reply by Conan179** [reply]
Ich hab keinen schimmer, wie man das anpasst.

**Reply by fda77** [reply, marked as answer]
Und ich keine Gerät zum testen
und keine Zeit
und keine Lust

**Reply by Conan179** [reply]
Du könntest schreiben was du bräuchtest, ich shcicke es dir und ich teste es dann

**Reply by fda77** [reply]
Einen Kabelanschluss und eine 6690
Das würde mich dann betreffen und meine Motivation wäre wesentlich höher

**Reply by Conan179** [reply]
und eine 6591, eine 7530

**Reply by Prisrak1** [reply]
> Einen Kabelanschluss und eine 6690 Das würde mich dann betreffen und meine Motivation wäre wesentlich höher

Wie lautet denn deine Anschrift? Ich bin bereits mit dem Kabel und einer passender Fritzbox zu dir unterwegs.

**Reply by fda77** [reply]
Warum machst dus nicht selbst wenn du die Teile herumliegen hast???

**Comment by reimerp** [comment]
Update, obwohl das schon relativ alt ist:
8.03 und 8.10 funkfionieren auch **nicht** vernünftig, haben den DECT error counter, der irgendwann die Box nestartet.
das dect issue ist in 8.20, deutlch besser. Kein Knacken mehr beim Telefonieren oder im Internetradio. Aber ich hab dennoch schon mehrere Reboots gehabt mit gleichem Log und das obwohl DECT zu der Zeit gar nicht genutzt wurde. Leider geht on 8.20 das Lösen eines ethernet ports aus dem Switch nicht nehr. bzw schon aber nicht per ar7.cfg. die Ports landen in der falschen bridge nich im lan

**Comment by reimerp** [comment]
8.25 ist ja raus: Test mit 5 Min Internetradio via DECT: Aussetzer nach wie vor da ca alle 20-30 Sekunden, logs dementsprecehen auch, ABER der error counter ist scheinbar ausgebaut, d.h. die Box bootet nicht mehr wenn diese einen Schwellwert überschritten haben, so noch passiert in 8.21

**Comment by fda77** [comment]
Dann kann es nciht mehr lange dauern bis avm es in den griff bekommt...
